### ISSUE TITLE
why my DeepLabCut train max epoch is 1000?

### ISSUE THREAD

**Issue by acer1204** [DeepLabCut/DeepLabCut#3227]
### Is there an existing issue for this?

- [x] I have searched the existing issues

### Operating System

win11

### DeepLabCut version

3.0.0rc13

### What engine are you using?

pytorch

### DeepLabCut mode

single animal

### Device type

GTX1060

### Bug description 🐛

train max epoch is 1000
display iterations max is 100000.
in older veriosn looks the epoch can set like 50000 or higher.

### Steps To Reproduce

1.use pycharm to run DeepLabCut
2.input 5s video
3.use label
4.train config

### Relevant log output

```shell

```

### Code of Conduct

- [x] I agree to follow this project's [Code of Conduct](https://github.com/DeepLabCut/DeepLabCut/blob/master/CODE_OF_CONDUCT.md)

**Comment by MMathisLab** [comment]
Hi, can you expand on what you experience; you set it to a higher # and it stops after 1K? Can you show me how you changed the max iterations?  (Also note in torch you don't need to train as long)

**Comment by acer1204** [comment]
> Hi, can you expand on what you experience; you set it to a higher # and it stops after 1K? Can you show me how you changed the max iterations? (Also note in torch you don't need to train as long)
In GUI, the config up to 1000. and can not higher.

in offical video.  the maximum iterations looks can set 100000?
save interations can set 50000
https://youtu.be/tr3npnXWoD4?t=380

<img width="912" height="486" alt="Image" src="https://github.com/user-attachments/assets/b9db9491-76df-47a8-8218-b86c7f22218a" />

**Comment by MMathisLab** [comment]
For TensorFlow backend, not recommended in PyTorch ... epochs and iterations are different. 

Your screenshot does not show a bug, can you please describe what the bug is? Can you not type a different #?

**Comment by acer1204** [comment]
> For TensorFlow backend, not recommended in PyTorch ... epochs and iterations are different.
> 
> Your screenshot does not show a bug, can you please describe what the bug is? Can you not type a different #?

yes,
usb keyboard can not input upper 1000. and press add button have same problem.
like this my record video.
https://github.com/user-attachments/assets/9ae5d109-c5de-4405-9544-1475735732b7

**Comment by C-Achard** [comment]
Hello @acer1204, indeed these values are hardcoded maximums in the UI (see [here](https://github.com/DeepLabCut/DeepLabCut/blob/2c7d8aaa9f4fe51a213b5a26dc51b358b0f8749b/deeplabcut/gui/tabs/train_network.py#L326) and [here](https://github.com/DeepLabCut/DeepLabCut/blob/2c7d8aaa9f4fe51a213b5a26dc51b358b0f8749b/deeplabcut/gui/tabs/train_network.py#L344)).

Just to recap, in the DLC GUI: 

- "Display iterations/epochs" controls how often the training progress prints (TF counts in iterations, PT in batches). 
- For PyTorch, maximum epochs is how many times the model training will go over the entire training data before stopping. 
- For TensorFlow this corresponds to each step over mini-batches, here called "iterations". This happens significantly more often, hence the difference in magnitude.

The docs recommend starting around 10k iterations for TF or **~200 epochs for PT** and adjusting from there.
You can do so based on whether the loss plateaus after those initial 200 epochs. If it doesn't, resume training for more epochs.

For Display Iterations, you can log it as often as you'd like by lowering it, but if you set it to the max it will log very infrequently if at all.
I'd recommend setting it every 10-100 batches (depending on your data), if you do want summaries per batch. 
Alternatively, you could also keep it high and rely on the epoch logging instead.

**Comment by C-Achard** [comment]
Closing for now, please re-open if needed @acer1204 !
